_DEPRECATED.travis.yml

```diff
@@ -2,15 +2,15 @@ os:
   - linux
 language: node_js
 node_js:
-  - '10.16.1'
+  - "10.16.1"
```

Comment:
why deprecated?

Reply:
According jhipster standard, now all the build will be on github actions. So I have renamed the travis file to avoid to start automatically build also on travis.

Reply:
This module supports projects built with JHipster.
However, the coding/ci standards are independent of that project.

Reply:
Now, if you generate a new module with the generator-jhipster-module, it is supported only github actions for ci